specs/flipper/flipper.md

```diff
@@ -0,0 +1,103 @@
+# Flipper
+
+## Overview
+
+The flipper component is most often used to page through blocks of content or collections of ui elements.
+
+### Use Cases
+- Tony is looking for a new refrigerator. He goes onto his computer and navigates the browser to a site that sells appliances. He sees a list of refrigerators but not the one he wants so he clicks an arrow button (flipper) to show more. He sees a new GE Cafe that interests him and clicks on it which directs him to a purchase page.
+
+- Bobby is blind and is also looking for a new refrigerator. He goes onto his computer and navigates the browser to the same site that sells appliances. He tabs through a list of refrigerators until he finds the one that was recommended to him by a friend. Bobby is unaware of the flipper.
+
+### Non-goals
+- Other button implementations
+
+### Prior Art/Examples
+Limited prior art is available from major component libraries, however many brands and products leverage flippers on the front page of their sites. From a cursory search, the following prominently display either a carousel or scrolling region powered by "flippers":
+- Hulu
+- Netflix
+- Amazon
+- Microsoft
+
+---
+
+
+### API
+
+Extends FAST Element
+
+*Component Name*
+- `fast-flipper`
+
+*Attrs*
+- `direction` - enum, previous or next. Defaults to next
+- `hiddenFromAT` - The control is hidden from assistive technology. Defaults to true
+- `disabled` - the control is disabled. There are scenarios where a flipper would be visible and disabled
+- `label` - the accessible label for the control, exposed to AT when not hidden
+
+*Events*
+- `click` - mimics the click event of traditional buttons
+
+### Anatomy and Appearance
+
+```
+<template
+    role="button" 
+    aria-hidden=${x => x.hiddenFromAT}
```

Comment:
Can we just let the end user decide by adding aria-hidden themself or is there something we need to do internally in response to the hiddenFromAT property changing?

Reply:
Some of this depends on how we want to default it. If you default it to being exposed to assistive tech, then you need to drop the tabindex and add aria hidden in the event you want to hide it. If you default to hidden from assistive tech, the end user needs to add tabindex, override the internal aria-hidden behavior, and add the role of button. It's shown as persisting above, but I don't think we would add that if it were not exposed to AT. So, in a sense it's a swing of 3 properties on the implementors side to successfully transition between these states. I see a single prop to toggle this as a value add for empowering accessibility; however, we'd technically be empowering it as well by making someone handle it themselves.